client_plugin/drivers/shared/shared_driver.go

```diff
@@ -55,6 +58,8 @@ const (
 	sambaImageName = "dperson/samba"
 	sambaUsername  = "root"
 	sambaPassword  = "badpass"
+	fileShareName  = "public"
```

Comment:
This is "share1" as we discussed. I will move this to dockerops and use the name share1 once I rebase.